opm/core/wells/well_controls.cpp

```diff
@@ -142,7 +142,7 @@ well_controls_create(void)
 {
     struct WellControls *ctrl;
 
-    ctrl = malloc(1 * sizeof *ctrl);
+    ctrl = static_cast<WellControls*>(malloc(1 * sizeof *ctrl));
```

Comment:
possibly want to change to new (and delete) to get throws on alloc failure.

Reply:
I think that would require larger changes to this code part, the C code is not very exception safe... Is it not better to leave it as it is until we can delete it (which is hopefully not too far off)?

Reply:
sure. was just a gut reaction.